### DISCUSSION TITLE
Where is the lighting configuration file?

### DISCUSSION THREAD

**Discussion by mrzebaba1** [T-Troll/alienfx-tools#388] [Q&A] [answered]
Because I need to reinstall the Windows system frequently, I need to re-enter every keyboard key and then reset the lights after installing the system. I directly copy the entire Alien Fx program folder to the mobile hard drive and reinstall it. I copied it into the system, but it doesn’t seem to work. I still need to enter each key and set the lighting effects. How should I back up the configuration file? So that I can use it directly next time I reinstall the system.

**Comment by T-Troll** [comment, marked as answer]
Ohh... Seems like old-school skill of "reading" lost completely!

I have a tool among others. Named `alienfx-config`.
Run from CMD `alienfx-config backup`. It will provide some .reg files with all tools configuration into the same folder. Copy it. After re-install, open it all (it will require regedit), and this will restore your configuration. Run tools after.

**Reply by mrzebaba1** [reply]
Oh, thank you so much, I have backed up my configuration files
